app/js/home_controller.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+/* global SumoDB, nunjucks */
+
+(function(exports) {
+  var USERS_LIST_TMPL = 'users_list.html';
+
+  var top_helpers_container;
+
+  function load_top_helpers() {
+    var promise = SumoDB.get_top_helpers();
+
+    promise.then(function(users) {
+      if (users.length) {
+        top_helpers_container.classList.remove('hide');
```

Comment:
You could do an early return here. No need to do a nunjucks render if we have no users.
```js
if (!users.length) {
  return;
}
// unhide
// insert nunjucks
```